package.xml

```diff
@@ -10,26 +10,20 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend>cv_bridge</build_depend>
-  <build_depend>eigen_conversions</build_depend>
   <build_depend>image_geometry</build_depend>
   <build_depend>image_transport</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>sensor_msgs</build_depend>
   <build_depend>tf2_eigen</build_depend>
   <build_depend>tf2_ros</build_depend>
-  <build_depend>sensor_msgs</build_depend>
```

Comment:
removing sensor_msgs here and below because it's already listed in alphabetical order above